gulpfile.js

```diff
@@ -17,6 +17,11 @@ const config = manifest.babelBoilerplateOptions;
 const mainFile = manifest.main;
 const destinationFolder = path.dirname(mainFile);
 const exportFileName = path.basename(mainFile, path.extname(mainFile));
+const banner = ['/**',
+  '  * <%= manifest.name %> - <%= manifest.description %>',
```

Comment:
This is the format that is suggest by Gulp-header, maybe there is a more elegant way to do this?